implementations/elixir/ockam/ockam/lib/ockam/transport/uds/connection.ex

```diff
@@ -0,0 +1,49 @@
+defmodule Ockam.Transport.UDS.Connection do
+  use GenServer
+
+  alias Ockam.Router
+  alias Ockam.Wire
+
+  require Logger
+
+  def start_link(socket) do
+    GenServer.start_link(__MODULE__, socket)
+  end
+
+  def send_message(listener, message) do
+    GenServer.cast(listener, {:send_message, message})
+  end
+
+  defstruct [:socket]
+
+  @impl true
+  def init(socket) do
+    Logger.debug("Started connection handler")
+    {:ok, %__MODULE__{socket: socket}}
+  end
+
+  @impl true
+  def handle_info({:tcp, socket, data}, %__MODULE__{socket: socket} = state) do
```

Comment:
This module currently only supports TCP->Ockam messages.
How do we reply to those messages?

In other transport handler workers we were tracing the address of the worker in the message return route (see `Ockam.Transport.TCP.Handler` around line 70).

I see this is not yet a worker, this comment is to keep in mind when you make it a worker.